tensorflow/tsl/BUILD

```diff
@@ -1,4 +1,5 @@
-load("tsl.bzl", "if_google", "if_oss")
+load("tsl.bzl", "if_google", "if_oss", "enable_targets_flag")
+load("//third_party/llvm:setup.bzl", "supported_targets")
```

Comment:
This will probably require some internal adjustments, so we would need to find someone to work on this. Internally we don't use setup.bzl.

Reply:
I think initially I tried to do the reverse and have third_party/llvm:setup.bzl use supported_targets defined in the core (or tsl, I can't quite remember) but there was some dependency issue. I'd need to try again.